app/features-json/setting_json_controller.rb

```diff
@@ -36,7 +41,10 @@ class SettingJSONController < APIController
         Services.setting_service.reset_setting!(setting_key: params[:setting_key])
         return json({ status: :success })
       rescue SettingServiceKeyNotFoundError
-        halt(404, "`#{params[:setting_key]}` not found.")
+        json_error(
```

Comment:
changing from `404` to `400` is intentional, right?

Reply:
Ohh, it's not actually, good find. Do you think it should be 404?